### DISCUSSION TITLE
Multiple UE in srs setup?

### DISCUSSION THREAD

**Discussion by warlock1302** [herlesupreeth/docker_open5gs#334] [Q&A]
Hi @herlesupreeth,all,
I am using docker_open5gs for testing srsUE and srsRAN through zmq.
i would like to attach multiple srsUE(2 atleast) to srsRAN and hence Open5gs core. 
could you please let me know if this is feasible and if so kindly help me with the process.
I could see ue_zmq.conf has one UE details and i am not sure how to add more.
Thanks,
Abhishek VJ
Research assistant
Techniche University Chemnitz

**Comment by herlesupreeth** [comment]
> could you please let me know if this is feasible and if so kindly help me with the process.

yes, its possible. With the current way in which I create multiple entities of same type (e.g. srsUE, SMF etc) its not very good. I will try to come up with a way over the weekend where its easy to deploy multiple entities. Will let you know once done

**Reply by ahmadhassandebugs** [reply]
I was wondering if there has been any update on this? Or, if you could suggest a way to implement this feature.

**Reply by herlesupreeth** [reply]
Sadly, no update on this. However, you can take a look at this commit - https://github.com/herlesupreeth/docker_open5gs/commit/ca0d937688c302c559a29cdfc5f4a61ef49cb335 where I introduce two SMFs and two UPFs to realize slicing scenario.
